docs/semgrep-app/triage-comment.md

```diff
@@ -0,0 +1,59 @@
+---
+slug: triage-comment 
+append_help_link: true
+title: Triaging findings through comments
+hide_title: true
+description: "Triage your findings in GitHub using comments."
+tags:
+    - Semgrep App
```

Comment:
For tags we want to indicate the following:
* If it is available on a free tier
* The minimum paid tier it is available on.

For example:

```
tags:
    - Semgrep App
    - Community Tier
    - Team & Enterprise Tier
```

or

```
tags:
    - Semgrep App
    - Team & Enterprise Tier
```

The reason for this is we want the tags endpoints to contain all possible docs for those tiers, and tags are not additive. They are singular.

Reply:
This is available from Community Tier. So:

```suggestion
    - Semgrep App
    - Community Tier
    - Team & Enterprise Tier
```

Reply:
Yup, feel free to commit!